src/modules/repository/implementation/sequelize/repositories/paranet-repository.js

```diff
@@ -0,0 +1,31 @@
+class ParanetRepository {
+    constructor(models) {
+        this.sequelize = models.sequelize;
+        this.model = models.paranet;
+    }
+
+    async getParanet(paranetId) {
```

Comment:
Please update the name of the function. By the name of the function i would expect that it will just get me Paranet, but it will also create new one if it doesn't exist. So the proper naming is something like: getOrCreateParanet or findOrCreateParanet